R/add_gnomAD_AF.R

```diff
@@ -85,3 +64,64 @@ add_gnomAD_AF <- function(data,
   mafdb
 }
 
+#' @rdname add_gnomAD_AF
+#'
+#' @examples
+#' \dontrun{
+#' BiocManager::install("MafDb.gnomAD.r2.1.hs37d5")
+#' file <- system.file("extdata", "allelic_counts_HG00187.csv", package = "tMAE", mustWork = TRUE)
+#' maeCounts <- fread(file)
+#' maeRes <- DESeq4MAE(maeCounts)
+#' genome_assembly <- 'hg19'
+#' res <- add_gnomAD_AF(maeRes, genome_assembly = genome_assembly, pop="AF")
+#' }
+#'
```

Comment:
I would keep the documentation all together when setting the genetic function. In addition, you rather use the exome data and run the example. This way we make sure the examples work.